test/parallel/test-fs-opendir.js

```diff
@@ -205,3 +210,20 @@ for (const bufferSize of ['', '1', null]) {
   assertDirent(dir.readSync());
   dir.close();
 }
+
+// Check that when passing a string instead of function - throw an exception
+async function doAsyncIterInvalidCallbackTest() {
+  const dir = await fs.promises.opendir(testDir);
+``
```

Comment:
Is this a typo?

Reply:
I'm sorry, yes it's a typo. I've removed it.

Reply:
fixed in https://github.com/nodejs/node/pull/30310/commits/b3e84a67b176febc061cc2b34dfd7c5b25a32220